### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by choskim**
- Updates text in README
- Links to examples that work

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,51 +1,99 @@
...
-    var rotation = Math.round(e.rotation);    
-    stage.style.transform = 'rotate('+rotation+'deg)';
+// Subscribe to desired event
+manager.on('rotate', function(e) {
+    var rotation = Math.round(e.rotation);  
+   
+    square.style.transform = 'rotate(' + rotation + 'deg)';
+    console.log("You're rotating me!");  
 });
 ```
```
Comment (jtangelder) [inline README.md]: Maybe also inform about the 'quick usage', just `new Hammer()` and you're good to go.

File: README.md [hunk 2]
```diff
@@ -1,51 +1,99 @@
...
-mc.on('rotate', function(e) {
-    // do something cool
-    var rotation = Math.round(e.rotation);    
-    stage.style.transform = 'rotate('+rotation+'deg)';
+// Subscribe to desired event
+manager.on('rotate', function(e) {
+    var rotation = Math.round(e.rotation);  
+   
+    square.style.transform = 'rotate(' + rotation + 'deg)';
+    console.log("You're rotating me!");
```
Comment (jtangelder) [inline README.md]: Log `e`, so the user can start discovering the properties.

**Comment by jtangelder** [comment]
Awesome, great improvement! Could you fix the nits I had?

**Comment by choskim** [comment]
@jtangelder: This has been updated to address your comments.

**Comment by jtangelder** [comment]
Thanks!

**Pull request merged**